WordPress/Docs/PHP/DiscouragedPHPFunctionsStandard.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0"?>
+<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd"
+    title="Discouraged PHP Functions"
+    >
+    <standard>
+    <![CDATA[
+Use JSON instead of serialized data, which has known vulnerability problems with object injection.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Using JSON for serialized data.">
+        <![CDATA[
+$serialized = <em>json_encode</em>( $array );
+$serialized = <em>wp_json_encode</em>( $array );
+$unserialized = <em>json_decode</em>( $array );
+        ]]>
+        </code>
+        <code title="Invalid: Using serialized data strings.">
+        <![CDATA[
+$serialized = <em>serialize</em>( $array );
+$unserialized = <em>unserialize</em>( $array );
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+URLs should now be encoded using rawurlencode(). Only legacy applications should use urlencode().
```

Comment:
Maybe it is not necessary to include the second sentence? The sniff will trigger anyway if it finds `urlencode()`. If there is a reason to include the second sentence, maybe it makes sense to add another sentence explaining what determines the cases where legacy applications should use `urlencode()`. I don't remember off the top of my head.